src/keyring/KeyringController.ts

```diff
@@ -282,6 +282,16 @@ export class KeyringController extends BaseController<
     }
   }
 
+  /**
+   * Method to validate a password against the password from the keyring.
+   *
+   * @param password - Password of the keyring.
+   * @returns Boolean indicating if input password is valid
+   */
+  validatePassword(password: string): boolean {
```

Comment:
Just one small nit pick
There are a couple of those places in the app where has this exact logic. Should we replace those places with this?

Reply:
Yep [2e83073](https://github.com/MetaMask/controllers/pull/884/commits/2e83073335793f4ccfa3c7a898cd5cd8db32053f)